domain-services/fiscal-event-service/fiscal-event-service-0.1.0.yaml

```diff
@@ -0,0 +1,374 @@
+---
```

Comment:
Please remove any changes to the file outside of the aggregator project.

Reply:
haven't done any changes in this file. while taking pull from develop and got conflict and accepted the develop branch one.